pages.ml/linux/pacman.md

```diff
@@ -0,0 +1,35 @@
+# pacman
+
+> ആർച്ച് ലിന്ക്സിന്റെ പാക്കേജ് മാനേജുമെന്റ് യൂട്ടിലിറ്റി.
```

Comment:
I don't speak the language, but I'm not sure about this translation:
![Screenshot_20201010_170645](https://user-images.githubusercontent.com/1536740/95664095-00c70e80-0b1b-11eb-91fb-35a06ea94312.png)
I found `ലിനക്സ്` is the word for Linux in https://olam.in/Dictionary/en_ml/linux
```suggestion
> ആർച്ച് ലിനക്സ് പാക്കേജ് മാനേജുമെന്റ് യൂട്ടിലിറ്റി.
```

Reply:
Yes, that is correct. But that would make it a word by word translation of the english version. In malayalam, instead of `ലിനക്സ് ` I used `ലിന്ക്സിന്റെ` to denote `linux's` because I felt it is more in line with the grammar of the language.